### PR TITLE
fuse: change download url

### LINKED ISSUE #4521: Fuse /packages/sysutils/fuse/package.mk
Source has moved from sourceforge see http://sourceforge.net/projects/fuse/files/?source=navbar

Any change of updating the package.mk file to point to the correct source?

### PR DESCRIPTION
**Pull request by CvH**
- no nice dl url due to github

- fixes #4521

### CODE REVIEW

File: packages/sysutils/fuse/package.mk [hunk 1]
```diff
@@ -21,8 +21,8 @@ PKG_VERSION="2.9.4"
 PKG_REV="1"
 PKG_ARCH="any"
 PKG_LICENSE="GPL"
-PKG_SITE="http://sourceforge.net/projects/fuse/"
-PKG_URL="$SOURCEFORGE_SRC/fuse/fuse-2.X/$PKG_VERSION/$PKG_NAME-$PKG_VERSION.tar.gz"
+PKG_SITE="https://github.com/libfuse/libfuse/"
+PKG_URL="https://github.com/libfuse/libfuse/releases/download/fuse_2_9_4/fuse-2.9.4.tar.gz"
```
Comment (MilhouseVH) [inline packages/sysutils/fuse/package.mk]: Untested, and might not work in `sh`, what about:

`PKG_URL="https://github.com/libfuse/libfuse/releases/download/${PKG_NAME}_${PKG_VERSION//./_}/$PKG_NAME-$PKG_VERSION.tar.gz"`?

Comment (CvH) [inline packages/sysutils/fuse/package.mk]: works too, if this is nicer - don't ask me :)

**Comment by chewitt** [comment]
might be nicer to point at our sources server and us an mkpkg_fuse script?

**Comment by CvH** [comment]
***If*** someone maintains it, but we have a working and stable dl url - the only -1 is that the dl url doesn't look pretty.

**Comment by sraue** [comment]
please use 
PKG_SITE="https://github.com/libfuse/libfuse/"
PKG_URL="https://github.com/libfuse/libfuse/releases/download/${PKG_NAME}_${PKG_VERSION//./_}/$PKG_NAME-$PKG_VERSION.tar.gz"

**Comment by sraue** [comment]
can you sqash both commits?

**Comment by CvH** [comment]
@sraue done

**Pull request merged**